packages/styles/README.md

```diff
@@ -73,6 +73,9 @@ import "@synergy-design-system/styles/typography.css";
 
 <!-- BEGIN INLINE COMMENT -->
 
+- table.css
+  - table-cell.css
+  - table.css
```

Comment:
I understand the problem here, however, it feels a bit redundant to have a filesystem layout like:

```bash
./dist
  ./table.css # Where the whole bundle of the module is
  ./table # Where the subs reside
    ./table-cell.css
    ./table.css # We may run into troubles when someone loads the wrong "table.css"
```

I do not have an idea currently how we can tackle this, but we should at least think about make this better in the future.